Libraries/Opc.Ua.Client/Subscription/Subscription.cs

```diff
@@ -76,29 +76,7 @@ public Subscription(Subscription template, bool copyEventHandlers)
 
             if (template != null)
             {
-                string displayName = template.DisplayName;
-
-                if (String.IsNullOrEmpty(displayName))
-                {
-                    displayName = m_displayName;
```

Comment:
I think the default handling when there is no default value in the template should be kept

Reply:
I was considering this (and can put that back), but then it's not a clone.

Reply:
<head scratching/> I guess there are pro and cons...

Reply:
ok, lets do a clone, otherwise really confusing